src/components/input/index.tsx

```diff
@@ -26,10 +32,12 @@ export const CustomInput = ({ label, password, placeholder, icon }: InputProps)
           required
           id={inputId}
           {...(placeholder && { placeholder })}
```

Comment:
É boa prática sempre fazer a verificação com uma `boolean`
```suggestion
          {...(!!placeholder && { placeholder })}
```